.gitignore

```diff
@@ -4,6 +4,7 @@ vision_videos/
 *.jpg
 *.png
 *.bag
+*.ipynb
```

Comment:
This should not be ignored, it is runnable code. Though you should reset the output of notebooks before commiting.